cordova-lib/src/platforms/platforms.js

```diff
@@ -60,6 +60,8 @@ function getPlatformApi(platform, platformRootDir) {
             events.emit('warn', ' Using this version of Cordova with older version of cordova-' + platform +
                 ' is being deprecated. Consider upgrading to cordova-' + platform + '@' +
                 platforms[platform].apiCompatibleSince + ' or newer.');
+        } else {
+            events.emit('warn', 'Error loading cordova-'+platform);
```

Comment:
Not a lot. However, whenever there is something wrong with the `PlatformApi`, the error that gets displayed to the user is indicating that they have the wrong version of the platform. In reality though, something could be wrong with the `PlatformApi` code which can be **super** misleading, hence the change. This is a very small change that can help immensely with debugging and troubleshooting errors related to `PlatformApi` code.